clang/lib/CIR/CodeGen/CIRGenExprScalar.cpp

```diff
@@ -87,6 +88,16 @@ static bool PromotionIsPotentiallyEligibleForImplicitIntegerConversionCheck(
   return SrcType->isIntegerType() && DstType->isIntegerType();
 }
 
+static bool isFPOrFPVectorTy(QualType CompType) {
```

Comment:
In the original LLVM codegen, this is done on top of LLVM types, not `QualType`, please implement this in terms of CIR type. Please also move this method close to `isAnyFloatingPointType`, as instructed in the previous review.

In this repo the comments are usually resolved by the reviewer, please don't mark them resolved yourself.